cmd/api-resources.go

```diff
@@ -71,7 +71,7 @@ func getListObjectsV2Args(values url.Values) (prefix, token, startAfter, delimit
 
 	// The continuation-token cannot be empty.
 	if val, ok := values["continuation-token"]; ok {
-		if len(val[0]) == 0 {
+		if len(val) == 0 {
```

Comment:
Maybe let's do this:
```
if len(val) == 0 || len(val[0]) == 0 
```

I am also not sure why the original code is not simply using values.Get("continuation-token")

Reply:
len(val[0]) is a byte.Here is `len(string)==0` ,I think. @vadmeste